rainier-tests/src/test/scala/com/stripe/rainier/compute/RealTest.scala

```diff
@@ -181,7 +181,11 @@ class RealTest extends FunSuite {
   run("x^(10(2/10 + 1/10))") { x =>
     x.pow((Real.one / 10 + Real.two / 10) * 10)
   }
-
+/*
+  run("x^((4/3 + 1)*3)") { x =>
```

Comment:
This fails even with BigDecimal, but we might be able to get it to pass with a different `Decimal` implementation